packages/underlay/test/underlay.test.ts

```diff
@@ -14,8 +14,17 @@ import { elementUpdated, expect, fixture, html } from '@open-wc/testing';
 
 import '../sp-underlay.js';
 import { Underlay } from '../';
+import { testForLitDevWarnings } from '../../../test/testing-helpers.js';
 
 describe('Underlay', () => {
+    testForLitDevWarnings(
+        async () =>
+            await fixture<Underlay>(
+                html`
+                    <sp-underlay></sp-underlay>
+                `
+            )
```

Comment:
For all of these calls, could you get away with omitting the wrapped `async` & `await`, since fixture is already returning a Promise? eg:

```suggestion
        () => fixture<Underlay>(
                html`
                    <sp-underlay></sp-underlay>
                `
            )
```

Reply:
Any chance that would be faster? I was going for uniformity, and by that I mean "copy and paste"-ability.

Reply:
I'm pretty sure it wouldn't be measurably faster. Just changes where the await happens (inside the fn or outside).